src/projects.js

```diff
@@ -35,35 +33,12 @@ export const ProjectList = () => (
 
 export const ProjectEdit = () => (
     <Edit>
-        <SimpleForm>
-            <TextInput disabled source="id" />
-            <ReferenceInput source="clientId" reference="clients">
-                <SelectInput optionText="name" />
-            </ReferenceInput>
-            <ReferenceInput source="projectTypeId" reference="project-types">
-                <SelectInput source="name" />
-            </ReferenceInput>
-            <TextInput source="name" />
-            <DateInput source="startDate" />
-            <DateInput source="endDate" />
-            <RichTextInput source="notes" />
-        </SimpleForm>
+        <ProjectFormLayout referenceValuesC={referenceValuesC} referenceValuesP={referenceValuesP} />
```

Comment:
i think we could change referenceValuesC and referenceValuesP to better names, like projectReferences and clientReferences

Reply:
Done!